modules/installation-special-config-encrypt-disk-tang.adoc

```diff
@@ -72,7 +83,7 @@ EOM
 ewogInVybCI6ICJodHRwczovL3RhbmcuZXhhbXBsZS5jb20iLAogInRocCI6ICJaUk1leTFjR3cwN3psVExHYlhuUWFoUzBHdTAiCn0K
 ----
 
-. Replace the “source” in the TPM2 example with the Base64 encoded file for the type of node to update:
+. Replace the “source” in the TPM2 example with the Base64 encoded file for the type of node to update, while also adding the kernel argument `rd.neednet=1`:
```

Comment:
You could potentially replace ", while also" with "and".

Reply:
On further review I think maybe this is two steps merged into one. I think the `rd.neednet=1` kernel argument, along with any other network related kernel arguments, should be introduced in a step of its own. Perhaps step 3 should be moved down after this one and have its own `.Example output`?

Reply:
I agree, and this is a better solution. I've introduced the new karg as a separate step + example after this one. Thanks!